### ISSUE TITLE
[Bug] Content Type text/xml not present

### ISSUE THREAD

**Issue by 2E0PGS** [getinsomnia/insomnia#649]
## [Bug] 
- Insomnia Version: v5.12.4
- Operating System: Windows 10 && Ubuntu 16.04.2

## Details
When selecting body type XML it should have option for text/xml and not just application/xml.
![image](https://user-images.githubusercontent.com/15659301/33680491-ad180486-dab9-11e7-8040-44fbb5f4b24e.png)

![image](https://user-images.githubusercontent.com/15659301/33680500-b4d98b22-dab9-11e7-81f1-9dc29cc3d686.png)

Some SOAP API require text/xml instead of application/xml. This auto generated header which I didn't realise for a while was application instead of text drove me up the wall for a while before I found the cause of the problem. 

So in summary option for auto generated header as text/xml instead of application/xml. 

Perhaps have two body distinctions for XML. Application/XML and Text/XML body each auto generate separate header Content-Type.

Also the intelisense / suggestion should also have suggestion for text/xml because it currently does not!

![image](https://user-images.githubusercontent.com/15659301/33680566-eedda524-dab9-11e7-806e-219d8035a118.png)
